torchaudio/_internal/module_utils.py

```diff
@@ -33,3 +34,19 @@ def wrapped(*args, **kwargs):
                 raise RuntimeError(f'{func.__module__}.{func.__name__} requires {req}')
             return wrapped
     return decorator
+
+
+def deprecated(direction: str):
```

Comment:
It give the consistent message format. We are going to deprecate a bunch of functions related to the current sox backend such as `sox_signalinfo_t` so it will make it easier to do the job.